src/main/java/net/dv8tion/jda/api/entities/ApplicationTeam.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2015-2019 Austin Keener, Michael Ritter, Florian Spieß, and the JDA contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.dv8tion.jda.api.entities;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import java.util.List;
+
+/**
+ * Meta-data for the team of an application.
+ */
+public interface ApplicationTeam extends ISnowflake
+{
+    /** Template for {@link #getIconUrl()} */
+    String ICON_URL = "https://cdn.discordapp.com/team-icons/%s/%s.png";
+
+    /**
+     * The id hash for the icon of this team.
+     *
+     * @return The icon id, or null if no icon is applied
+     *
+     * @see    #getIconUrl()
+     */
+    @Nullable
+    String getIconId();
+
+    /**
+     * The url for the icon of this team.
+     *
+     * @return The icon url, or null if no icon is applied
+     */
+    @Nullable
+    default String getIconUrl()
+    {
+        String iconId = getIconId();
+        return iconId == null ? null : String.format(ICON_URL, getId(), iconId);
+    }
+
+    /**
+     * The {@link net.dv8tion.jda.api.entities.TeamMember Team Members}.
+     *
+     * @return Immutable list of team members
+     */
+    @Nonnull
+    List<? extends TeamMember> getMembers();
```

Comment:
This bounded wildcard makes `list.add(new TeamMember())` a compile-time error but its probably not worth it to break our consistency for it.